Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_-_Quarantine_File.yml

```diff
@@ -0,0 +1,427 @@
+        "position": {
+          "x": 450,
+          "y": 380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 410c04ef-0afc-4578-8b1b-b1a3de96debf
+    type: condition
+    task:
+      id: 410c04ef-0afc-4578-8b1b-b1a3de96debf
+      version: -1
+      name: Was the file quarantined?
+      description: Checks if the quarantining of the file was successful.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "16"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEqualString
+          left:
+            value:
+              complex:
+                root: Core.GetActionStatus
+                accessor: status
+            iscontext: true
+          right:
+            value:
+              simple: FAILED
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: Core.GetActionStatus
+                accessor: status
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 755
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 7e2c6273-86db-4fdd-8165-79a451736b14
+    type: regular
+    task:
+      id: 7e2c6273-86db-4fdd-8165-79a451736b14
+      version: -1
+      name: Set quarantine files to the context
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      key:
+        simple: QuarantinedFilesFromEndpoints
+      value:
+        complex:
+          root: Core.quarantineFiles
+          accessor: actionIds
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 980
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "13_2_#default#": 0.14,
+      "15_16_yes": 0.6,
+      "15_2_#default#": 0.23
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1360,
+        "width": 380,
+        "x": 450,
+        "y": 70
+      }
+    }
+  }
+inputs:
+- key: FileContainment
+  value:
+    simple: "True"
+  required: false
+  description: Set to 'True' to quarantine the identified file.
+  playbookInputQuery:
+- key: FileRemediation
```

Comment:
Maybe we should change it to FileRemediationType ?

Reply:
It is affecting all the playbooks that utilize the containment plan as a sub-playbook